### DISCUSSION TITLE
Pdfs files not opening

### DISCUSSION THREAD

**Discussion by negisuraj080** [community/community#72774] [Repositories]
### Select Topic Area

Bug

### Body

Seeing this issue when trying to open pdfs stored on github "Unable to render code block". I am using chrome browser. tried clearing cache but the issue is still there. Its this same issue. https://github.com/orgs/community/discussions/64419

![image](https://github.com/community/community/assets/143009358/db4d0ee7-81ba-4ba7-80e3-1f941f8cc643)

**Comment by ktauchathuranga** [comment]
try using incognito tab

**Comment by negisuraj080** [comment]
In incognito mode its still not working

**Comment by arienshibani** [comment]
Try to hold down Ctrl and press F5 (force chrome to empty cache and refresh window). If it **does not** fix the issue, there must be something wrong with your web browser (because the PDF in question displays perfectly fine in my browser Version 118.0.5993.70). 

I suspect you might have an outdate version of chrome? Check what version you are on, and see if an updates fixes things.

**Comment by negisuraj080** [comment]
my chrome is updated still facing the same issue
